### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by benwmx**
the first branch of the project contains config files and the skeleton of the mobile version of my portfolio.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Roboto+Condensed:ital,wght@1,700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <nav class="navbar">
+       <p class="nav-item">Rb</p>
+       <img class="nav-item" src="./img/menu.png" alt="the menu image ">
+    </nav>
```
Comment (carlos-ssh) [inline index.html]: - Nav tag should be inside the header.
Marked in the first steps in your requirements project(The container of the section is a header tag).

File: index.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        <p class="secondary">I'm a software developer</p>
+        <p class="details">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.
+        
+        </p>
+        <div class="social-media">
+            <a href="https://github.com/benwmx" target="_blank"><img src="./img/002-github-big-logo.png" alt="the image ogf github icon"></a>
+            <a href="https://www.linkedin.com/in/rachid-boudaoudi-1621a0183/"target="_blank"><img src="./img/001-linkedin.png" alt="the image of linkedin icon"></a>
+            <a href="https://twitter.com/benwmx" target="_blank"><img src="./img/003-twitter.png" alt="the image of twitter icon"></a>
+        </div>
```
Comment (carlos-ssh) [inline index.html]: - Your code should be inside a section tag. (1.- The container of the section is a section tag)

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @benwmx, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

‼️
**_Kindly check that all your linters are passing(green color) in your pull request, before to create another code review._**
‼️


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Emmyn5600 — APPROVED** [review]
## STATUS: Approved :clap: :clap:

### Hi @benwmx :raised_hands:

Congratulations!:tada:
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Great work on implementing the changes requested by the previous code reviewer.

###  Optional but I highly recommend you to work on it before you merge it.

- [ ] Your screenshot image in the Readme file is not working, please fix it.
- [ ] Include the descriptive title of your project instead of keeping Document as a title.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**